public/src/components/admin/CreateCourse.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { useState } from "react"
+import { Course } from "../../../proto/qf/types_pb"
+import { useActions, useAppState } from "../../overmind"
+import CourseForm from "../forms/CourseForm"
+import CourseCreationInfo from "./CourseCreationInfo"
+import { Color } from "../../Helpers"
+
+
+const CreateCourse = (): JSX.Element => {
+    const state = useAppState()
+    const actions = useActions()
+    const [course, setCourse] = useState<Course>()
+    const [orgName, setOrgName] = useState("")
+    
+    const refresh = React.useCallback(async () => {
+        await actions.getCourses()
+        const c = state.courses.find(c => c.ScmOrganizationName === orgName)
+        if (c) {
+            await actions.getEnrollmentsByUser()
+            setCourse(c)
+        } else {
+            actions.alert({ text: "Course not found", color: Color.YELLOW, delay: 5000 })
+        }
+    }, [orgName])
+
```

Comment:
VSCode complains on this line with this message:
```
React Hook React.useCallback has missing dependencies: 'actions' and 'state.courses'. Either include them or remove the dependency array.eslintreact-hooks/exhaustive-deps
const orgName: string
```
The QuickFix suggest making this line be something like:
```tsx
    }, [actions, orgName, state.courses])
```
There were other QuickFix suggestions as well.